### DISCUSSION TITLE
コメント&ディスカッションで発言者の明示

### DISCUSSION THREAD

**Discussion by RyoyaC** [code-application/reading-notebook#3] [General]
ノート内のコメントおよびディスカッションで後で読んだときどっちの意見かぱっとわからなくなる可能性があるので、文末に(@RyoyaC )のように発言者を明示するようにするのはどうですか？
- コミット履歴さかのぼればどっちが書いたかわかるかもしれないけど、わざわざそこまでするのは面倒くさいと思います
- 本の内容をまとめているところやTIPSはあえて記載しなくてもいいかと思っています

**Comment by Kattsuuya** [comment]
同意です！
そうしましょう。
今回分については次回集まったときに追記します！
